pkg/operator/ceph/controller/spec.go

```diff
@@ -813,6 +813,32 @@ func LogCollectorContainer(daemonID, ns string, c cephv1.ClusterSpec) *v1.Contai
 	}
 }
 
+// rgw operations will be log
+func RGWOpsLogContainer(daemonID, ns string, c cephv1.ClusterSpec) *v1.Container {
+
+	// Generate ops log file name based on daemon ID
+	opsLogFile := fmt.Sprintf("ops-log-%s.log", daemonID)
+
+	return &v1.Container{
+		Name: "rgw-ops-log",
+		Command: []string{
+			"/bin/bash",
+			"-x", // Print commands and their arguments as they are executed
+			"-e", // Exit immediately if a command exits with a non-zero status.
+			"-m", // Terminal job control, allows job to be terminated by SIGTERM
+			"-c", // Command to run
+			fmt.Sprintf("tail -n+1 -F %s", path.Join(config.VarLogCephDir, opsLogFile)),
```

Comment:
Is it necessary to use bash like `bash -c 'tail ...'` instead of just `tail ...` directly?